dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -274,9 +270,20 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
                     THROW_HR_IF(E_INVALIDARG, s_protocolRegistration);
                 }
 
+                s_taskClsid = details.TaskClsid();
                 wil::unique_cotaskmem_string unpackagedAppUserModelId;
                 RegisterUnpackagedApplicationHelper(GUID_NULL, unpackagedAppUserModelId); // create default registration for app
 
+                if (AppModel::Identity::IsPackagedProcess())
```

Comment:
We should talk about this. In general, Dveloper should be allowed to use the PushTrigger flag in conjunction with the COM activator flag for all packaged apps irrespective of whether we run in RS5 or not.

Reply:
And for unpackaged, the developer always uses ProtocolActivator flag.